README.md

```diff
@@ -2,58 +2,53 @@
 
 The `"github.com/newrelic/opentelemetry-exporter-go/newrelic"` package
 provides an exporter for sending OpenTelemetry data to New Relic.  Currently,
-traces and the latest metric instruments (as of v0.8 of Open Telemetry) are supported.
+traces and the latest metric instruments (as of v0.12 of Open Telemetry) are
+supported.
 
 Example use:
 
 ```go
-import (
-	"log"
-	"os"
+import "github.com/newrelic/opentelemetry-exporter-go/newrelic"
 
-	"github.com/newrelic/opentelemetry-exporter-go/newrelic"
-	"go.opentelemetry.io/otel/api/global"
-	"go.opentelemetry.io/otel/sdk/trace"
-)
-
-func initTracer() {
-	exporter, err := newrelic.NewExporter("My Service", os.Getenv("NEW_RELIC_API_KEY"))
-	if err != nil {
-		log.Fatal(err)
-	}
-	tp, err := trace.NewProvider(trace.WithSyncer(exporter))
+func main() {
+	// Assumes the NEW_RELIC_API_KEY environment variable contains your New
+	// Relic Insights insert API key. This will error if it does not.
```

Comment:
That Insights insert API has been rebranded to the "Event API". https://docs.newrelic.com/docs/telemetry-data-platform/ingest-manage-data/ingest-apis/use-event-api-report-custom-events